clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/BasicAmazonDeployDescription.groovy

```diff
@@ -80,5 +81,7 @@ class BasicAmazonDeployDescription extends AbstractAmazonCredentialsDescription
     String region
     String asgName
     Boolean useSourceCapacity
+
+    LaunchConfiguration launchConfiguration
```

Comment:
Agreed ... I'll move `buildBlockDeviceMappings` there and parameterize in the `sourceLaunchConfiguration`.